lib/kino/shorts.ex

```diff
@@ -0,0 +1,573 @@
+
+  ## Examples
+
+      download(fn ->
+        "Example text"
+      end)
+
+      download(
+        fn -> Jason.encode!(%{"foo" => "bar"}) end,
+        filename: "data.json"
+      )
+
+      download(
+        fn -> <<0, 1>> end,
+        filename: "data.bin",
+        label: "Binary data"
+      )
+
+  """
+  @spec download((-> binary()), keyword()) :: Kino.Download.t()
+  def download(content_fun, opts \\ []), do: Kino.Download.new(content_fun, opts)
+
+  @doc ~S'''
+  Renders Markdown content, in case you need richer text.
+
+  It is a wrapper around `Kino.Markdown.new/1`.
+
+  ## Examples
+
+      import Kino.Shorts
+
+      markdown("""
+      # Example
+
+      A regular Markdown file.
+
+      ## Code
+
+      ```elixir
+      "Elixir" |> String.graphemes() |> Enum.frequencies()
+      ```
+
+      ## Table
+
+      | ID | Name   | Website                 |
+      | -- | ------ | ----------------------- |
+      | 1  | Elixir | https://elixir-lang.org |
+      | 2  | Erlang | https://www.erlang.org  |
+      """)
+  '''
+  @spec markdown(String.t()) :: Kino.Markdown.t()
+  def markdown(markdown), do: Kino.Markdown.new(markdown)
+
+  @doc """
+  Renders plain text content.
+
+  It is similar to `markdown/1`, however doesn't interpret any markup.
+
+  It is a wrapper around `Kino.Text.new/1`.
+
+  ## Examples
+
+      import Kino.Shorts
+      text("Hello!")
+
+  """
+  @spec text(String.t()) :: Kino.Text.t()
+  def text(text), do: Kino.Text.new(text)
+
+  @doc ~S'''
+  `Kino.Mermaid` renders Mermaid graphs:
+
+  It is a wrapper around `Kino.Mermaid.new/1`.
+
+  ## Examples
+
+      import Kino.Shorts
+
+      mermaid("""
+      graph TD;
+        A-->B;
+        A-->C;
+        B-->D;
+        C-->D;
+      """)
+  '''
+  @spec mermaid(String.t()) :: Kino.Mermaid.t()
+  def mermaid(mermaid), do: Kino.Mermaid.new(mermaid)
+
+  @doc """
+  A placeholder for static outputs that can be dynamically updated.
+
+  The frame can be updated with the `Kino.Frame` module API.
+  Also see `Kino.animate/3`.
+
+  ## Examples
+
+      import Kino.Shorts
+      frame = frame() |> Kino.render()
+
+      for i <- 1..100 do
+        Kino.Frame.render(frame, i)
+        Process.sleep(50)
+      end
+
+  """
+  @spec frame(keyword()) :: Kino.Frame.t()
+  def frame(opts \\ []), do: Kino.Frame.new(opts)
+
+  @doc """
+  Displays arbitrarily nested data structure as a tree view.
+
+  It is a wrapper around `Kino.Tree.new/1`.
+
+  ## Examples
+
+      import Kino.Shorts
+      tree(Process.info(self()))
+
+  """
+  @spec tree(term()) :: Kino.Layout.t()
+  def tree(tree), do: Kino.Tree.new(tree)
+
+  @doc ~S'''
+  Displays arbitrary static HTML.
+
+  It is a wrapper around `Kino.HTML.new/1`.
+
+  ## Examples
+
+      import Kino.Shorts
+
+      html("""
+      <h3>Look!</h3>
+
+      <p>I wrote this HTML from <strong>Kino</strong>!</p>
+      """)
+
+  '''
+  @spec html(String.t()) :: Kino.HTML.t()
+  def html(html), do: Kino.HTML.new(html)
```

Comment:
Shouldn't all of them be `defdelegate`?

Reply:
the delegate version with `:as` ends up being more verbose, not worth it IMO.

Reply:
Ah it's the same number of dispatches, nevermind!